examples/python-runtime-sandbox/tester.py

```
@@ -207,6 +208,38 @@ def test_upload(base_url):
+    script = (
+        "import pandas as pd, lightgbm as lgb; "
+        "from sklearn.linear_model import LinearRegression; "
+        "m = LinearRegression().fit([[0], [1]], [0, 1]); "
+        "print('ml-ok', int(round(m.predict([[2]])[0])))"
+    )
+    payload = {"command": f"python -c {shlex.quote(script)}"}
+
+    try:
+        print(f"\n--- Testing ML libraries ---")
+        print(f"Sending POST request to {url}")
+        response = requests.post(url, json=payload)
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)